index.html

```diff
@@ -54,6 +54,8 @@
       <div class="grid_2 omega section">
         <p class="title">apoio</p>
         <a href="http://codeminer42.com" rel="nofollow" target="_blank">Codeminer42</a>
+        <br />
+        <p><a href="file:///home/andrew/gurusorocaba.github.io/index.html">Pt-br</a> / <a href="file:///home/andrew/gurusorocaba.github.io/en.html">En</a></p>
```

Comment:
`file:///home/andrew/gurusorocaba.github.io/index.html` this is a local link, please use something like `/index.html`